docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md

```diff
@@ -0,0 +1,29 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Use one of the following approaches to link the process to be called by a call activity.
+---
+
+You can use one of the following approaches to link the process to be called by a [call activity](/components/modeler/bpmn/call-activities/call-activities.md).
+
+### Using the link button
+
+1. Select a call activity task from the canvas and a link button will appear at the bottom right.
+2. Click on the button and choose any diagram from the same project.
```

Comment:
I would specify what button you are referring to here. For example, "Click the link button and choose any diagram..."

Reply:
The next button also mentions clicking on the `Link` button which is a different link button. I didn't want to say "Click the link button on 2 consecutive lines, which could make it confusing.